inference-engine/tests/functional/inference_engine/transformations/pad_fusion.cpp

```diff
@@ -119,11 +119,11 @@ TEST(TransformationTests, PadFusionConvolution) {
 
     Shape data_shape{1, 3, 14, 14};
     {
```

Comment:
In these transformation tests the element types for inputs of convolution ops were changed from integer to float